lib/irc/ClientPool.js

```diff
@@ -204,6 +204,20 @@ ClientPool.prototype._getNumberOfConnections = function(server) {
     return numConnectedNicks;
 };
 
+ClientPool.prototype.countTotalConnections = function() {
+    var count = 0;
+
+    Object.keys(this._virtualClients).forEach((domain) => {
+        var clients = this._virtualClients;
```

Comment:
Er.. wow. I'm surprised this works. Indeed I did. ((also: I really want `Object.values()`))